static/app/views/starfish/utils/extractRoute.tsx

```diff
@@ -7,5 +7,8 @@ export function extractRoute(location: Location) {
   if (location.pathname.match(/^\/starfish\/database\//)) {
     return 'database';
   }
-  return 'endpoint-overview';
+  if (location.pathname.match(/^\/starfish\/endpoint-overview\//)) {
+    return 'endpoint-overview';
+  }
+  return 'spans';
```

Comment:
Would be safer to not return anything if there's no matching route!

Reply:
updated to return null and updated callers to handle null value